src/tests/Microsoft.VisualStudio.Composition.Tests/ByValueEqualityTests.cs

```diff
@@ -79,6 +79,17 @@ public void AssemblyNameComparer_ReturnsFalseWhenPublicKeysDontMatch()
             Assert.NotEqual(a1, a2, ByValueEquality.AssemblyName);
         }
 
+        [Fact]
+        public void AssemblyNameComparerNoFastCheck_SkipsCodeBaseCheck()
+        {
+            byte[] publicKey = GetPublicKeyFromExecutingAssembly();
+            AssemblyName a1 = CreateAssemblyName("AssemblyA", new Version(1, 0), CultureInfo.CurrentCulture, @"C:\some\path\AssemblyA.dll", publicKey: publicKey);
+            AssemblyName a2 = CreateAssemblyName("AssemblyA", new Version(2, 0), CultureInfo.CurrentCulture, @"C:\some\other\path\AssemblyA.dll");
```

Comment:
You also gave them different paths. So it seems you're not really testing the new functionality. Did you mean for these paths to be the same, since you're testing different assembly versions?